community/sway/etc/skel/.config/zsh/.zshrc

```diff
@@ -1,19 +1,16 @@
 # base config for oh my zsh
 source /usr/share/oh-my-zsh/zshrc
 
-# oh my zsh overrides
-plugins=(
-    archlinux
-    git
-)
-
-ZSH_THEME="agnoster"
-
-[ -d ~/.config/zsh/config.d/ ] && source ~/.config/zsh/config.d/*
-
-source $ZSH/oh-my-zsh.sh
```

Comment:
so now we disable omz completely?

Reply:
Nope, it's still loaded through `source /usr/share/oh-my-zsh/zshrc`, in the end of that file it sources OMZ. I had a great temptation to disable it, but unfortunately it leads to some weird glitches in foot (p10k maintainer says it's a terminal bug and it happens in iTerm2 and kitty too), I have lived with them on my own (I don't use OMZ), but decided not to torture all the people using this distro, so now it remains enabled, close to the Manjaro stock behavior. You can check if OMZ is loaded by doing `echo $ZSH_CACHE_DIR` - it sets this var on init.